ra/ra.go

```diff
@@ -2165,6 +2165,12 @@ func (ra *RegistrationAuthorityImpl) NewOrder(ctx context.Context, req *rapb.New
 	if err := ra.checkLimits(ctx, newOrder.Names, newOrder.RegistrationID); err != nil {
 		return nil, err
 	}
+	if !features.Enabled(features.CheckFailedAuthorizationsFirst) {
```

Comment:
```suggestion
	if features.Enabled(features.CheckFailedAuthorizationsFirst) {
```

Reply:
Good catch. Will fix. But now I'm wondering how this passed the test...

Reply:
Ah, it turns out the test was setting the wrong feature flag.